controllers/operands/kubevirt_test.go

```diff
@@ -2617,6 +2617,47 @@ Version: 1.2.3`)
 				Expect(kv.Spec.Configuration.HandlerConfiguration.RestClient.RateLimiter.TokenBucketRateLimiter.QPS).To(Equal(float32(100)))
 				Expect(kv.Spec.Configuration.HandlerConfiguration.RestClient.RateLimiter.TokenBucketRateLimiter.Burst).To(Equal(200))
 			})
+
+		})
+
+		Context("Tune KubeVirt rateLimiters with tuning profiles", func() {
+
+			var hco *hcov1beta1.HyperConverged
+
+			BeforeEach(func() {
+				hco = commonTestUtils.NewHco()
+			})
+
+			It("Should be empty by default", func() {
+				kv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(hco.Spec.TuningPolicy).To(BeEmpty())
+				Expect(kv.Spec.Configuration.APIConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.ControllerConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.WebhookConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.HandlerConfiguration).To(BeNil())
+
+			})
```

Comment:
We have the exact same test above. Please remove.

If you want, you can maybe wrap the annotation context and the highBurst context with additional context, and move this test to the common part. But I'm not sure it's not an overkill.

Reply:
Done. Check if it's better now. I've also added a new test covering the situation where the `hco.kubevirt.io/tuningPolicy` annotation is set but the profile is set to `highBurst`.